tools/tsp-client/src/index.ts

```diff
@@ -63,8 +64,10 @@ async function sdkInit(
     return newPackageDir;
   } else {
     // Local directory scenario
-    let configFile = joinPaths(config, "tspconfig.yaml")
-    const data = await readFile(configFile, "utf8");
+    if (!config.endsWith("tspconfig.yaml")) {
+      config = joinPaths(config, "tspconfig.yaml");
```

Comment:
Do we get a meaningful error if the file doesn't exist? If we don't get an actionable error from node we may want to catch the exception and rise a custom error to help customers

Reply:
Yeah I didnt see a good one returned from node...it was a ENOENT error will catch is on the line below and raise a better error for customers. Thanks for pointing that out!